### ISSUE TITLE
There is an selfi camera mode error in iphone.

### ISSUE THREAD

**Issue by toysun** [hiukim/mind-ar-js#400]
I tested your face module demo link by my iphone. but  iphone's camera mode is not selfi(user).  I tried to change camera mode by javascript, but also can't did it.

Pls check your face Tracking example in iphone. and pls teach me how to change camera mode.

**Comment by carlesgutierrez** [comment]
In my case I would like to active rear cámera for Android as default.
I've tried to switch camera with [switchCamera()](https://github.com/hiukim/mind-ar-js/blob/master/src/face-target/three.js) but this give me error. 
```
`Uncaught TypeError: Cannot read property 'srcObject' of undefined
    at U.stop (cdn.jsdelivr.net/npm/mind-ar@1.2.0/dist/mindar-face-three.prod.js:17)
    at U.switchCamera (cdn.jsdelivr.net/npm/mind-ar@1.2.0/dist/mindar-face-three.prod.js:22)
    at /animated-arface/moduleFaceMesh.js:27`
```
   
at mindarThree.switchCamera();

```
const {renderer, scene, camera} = mindarThree;
mindarThree.switchCamera();
```

**Comment by carlesgutierrez** [comment]
Did you tried with "mindarThree.switchCamera();" @toysun ? May be I'm doing it in the wrong place.

**Comment by hiukim** [comment]
should be fixed in v1.2.2

**Comment by toysun** [comment]
Thank you for your reply!

> should be fixed in v1.2.2
